### ISSUE TITLE
test.js does not produce output when Spotify and SpotifyWebHelper are open

### ISSUE THREAD

**Issue by roncli** [onetune/spotify-web-helper#3]
Running on Win10, I can't get the test.js to produce any output when Spotify is running and playing, and the SpotifyWebHelper executable are active.  I would expect .on("ready", ...) to be run and output the status of the player to the console, but am getting no output at all:

```
c:\Users\Ronald\Desktop\roncliGaming\webserver\node_modules\@jonny\spotify-web-helper>node test.js

c:\Users\Ronald\Desktop\roncliGaming\webserver\node_modules\@jonny\spotify-web-helper>
```

**Comment by roncli** [comment]
More information... this is caused by a bug in wintools, which is fixed by anodejs/node-wintools#3 but not yet published to npm... `node-spotify-webhelper` actually threw an error, where `@jonny/spotify-web-helper` just exits cleanly.

Not sure if this is something that you can do anything about without @anodejs publishing an update to `wintools` with the fix, but definitely something to be aware of.

**Comment by JonnyBurger** [comment]
I have included your newer version of `ps` and removed the dependency of `wintools`.

Both versions work for me on Windows.
